### PR TITLE
Add 3D Tiles Next links to extensions landing page

### PR DESCRIPTION
**Pull request by ptrgags**
Updated links to the extensions page for the new 3D Tiles Next extensions.

@donmccurdy could you review?

[Landing page](https://github.com/CesiumGS/3d-tiles/tree/update-extension-links/extensions)

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 ## Extensions
 
+### 3D Tiles Next Extensions:
```
Comment (donmccurdy) [inline extensions/README.md:5]:
```suggestion
### 3D Tiles Next
```

File: extensions/README.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+3D Tiles Next is a collection of draft extensions to advance 3D Tiles. It provides extensions for cleaner integration with the glTF ecosystem, rich semantic metadata, and efficient spatial indexes for scalable simulations.
+the [3D Tiles Next](../next) page for more details.
+
+* [3DTILES_content_gltf](./3DTILES_content_gltf)
+* [3DTILES_multiple_contents](./3DTILES_multiple_contents)
+* [3DTILES_metadata](./3DTILES_metadata)
+* [3DTILES_implicit_tiling](./3DTILES_implicit_tiling)
+* [3DTILES_bounding_volume_S2](./3DTILES_bounding_volume_S2)
+
+### Other Extensions:
```
Comment (donmccurdy) [inline extensions/README.md:16]:
```suggestion
### Other
```

Alternatively, would `3D Tiles 1.0` be better?

File: extensions/README.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
 
 ## Extensions
 
+### 3D Tiles Next Extensions:
+
+3D Tiles Next is a collection of draft extensions to advance 3D Tiles. It provides extensions for cleaner integration with the glTF ecosystem, rich semantic metadata, and efficient spatial indexes for scalable simulations.
+the [3D Tiles Next](../next) page for more details.
```
Comment (donmccurdy) [inline extensions/README.md:8]:
```suggestion
See the [3D Tiles Next](../next) page for more details.
```

File: extensions/README.md [hunk 4]
```diff
@@ -57,4 +70,4 @@ All extensions used in a tileset or any descendant external tilesets must be lis
 }
 ```
 
-All extensions required to load and render a tileset or any descendant external tilesets must also be listed in the tileset JSON in the top-level `extensionsRequired` array property, such that `extensionsRequired` is a subset of `extensionsUsed`. All values in `extensionsRequired` must also exist in `extensionsUsed`.
\ No newline at end of file
+All extensions required to load and render a tileset or any descendant external tilesets must also be listed in the tileset JSON in the top-level `extensionsRequired` array property, such that `extensionsRequired` is a subset of `extensionsUsed`. All values in `extensionsRequired` must also exist in `extensionsUsed`.
```
Comment (donmccurdy) [inline extensions/README.md:73]:
```suggestion
All extensions required to load and render a tileset or any descendant external tilesets must be listed in the tileset JSON in the top-level `extensionsRequired` array. Extensions in `extensionsRequired` must also be listed in `extensionsUsed`.
```

(Optional) The last sentence of this paragraph seemed redundant, so just rephrasing a bit.

**Review by donmccurdy — COMMENTED** [review]
A couple tweaks, otherwise LGTM!

File: extensions/README.md (hunk 2 above)
Comment (ptrgags) [reply extensions/README.md]: I think Other is better. Technically even 3D Tiles Next are 1.0 extensions

**Comment by ptrgags** [comment]
@donmccurdy updated!

**Review by donmccurdy — APPROVED** [review]
Looks good! 🚀

**Pull request merged**